base/shell/explorer/explorer.cpp

```diff
@@ -49,6 +49,41 @@ static VOID InitializeAtlModule(HINSTANCE hInstance, BOOL bInitialize)
     }
 }
 
+static BOOL
+GetServerAdminUIDefault()
+{
+    BOOL server;
+    DWORD value = 0, size = sizeof(value);
+    LPCWSTR rosregpath = L"SYSTEM\\CurrentControlSet\\Control\\ReactOS\\Settings\\Version";
```

Comment:
static const ?

Reply:
You remember we had this argument before? I don't have a problem with changing it here (even though the result is the same) but when you advice people to do this, you must say `static const WCHAR []?` You don't want anyone to write `static const LPCWSTR` because that is something else.